common/libs/wallet/myetherwallet.js

```diff
@@ -0,0 +1,353 @@
+import {
```

Comment:
I'm also not sure if its even worth doing the ES5 -> ES6 conversion, it adds to the diff between v3 and could serve to obfuscate a crypto change. 

Discussion is good before merging.